apps/src/p5lab/spritelab/commands/behaviorCommands.js

```diff
@@ -91,6 +91,9 @@ export const commands = {
   draggableFunc() {
     return spriteArg => {
       let sprite = this.getSpriteArray(spriteArg)[0];
+      if (!sprite) {
+        return;
+      }
```

Comment:
Nice! 
I have a question. On production, if I have a program with only the behavior begins being draggable block and run, there is no error. Do you happen to know why it doesn't end up at this point in this case? Thanks!

Reply:
`draggableFunc` is only executed on sprites that actively have the draggable behavior. If you use a `sprite begins` block when there are no matching sprites, the behavior doesn't get added to any sprites. Because no sprites are draggable, the function never needs to run. In the edge case you found, the sprites _were_ draggable, but they were also recently destroyed.